packages/react/ds/src/modal/modal.tsx

```diff
@@ -60,36 +82,37 @@ const ModalCloseButton = ({
   );
 };
 
-const isModalTitle = (child: any): boolean => {
-  if (!isValidElement(child)) {
-    return false;
-  }
-  const childType = child.type as any;
-  // @ts-expect-error The TS error says there is no _owner but there is
-  return childType === ModalTitle || child?._owner?.name === 'ModalTitle';
-};
-
 export const ModalWrapper = ({
+  position = 'center',
```

Comment:
We should let the user be able to define the modal width (sm, md, lg).
I don't see that property